source/element_parameters.md

```diff
@@ -0,0 +1,225 @@
+  7) End at downstream end of element.
+```
+
+### material
+
+The `material` parameter sets the material of the aperture. 
+Using chemical formulas like `Cu` and `Fe` are the most portable.
+
+### wall2d
+
+The `WALL2D` setting for `aperture_shape` is for defining an aperture using a 
+set of vertex points as illustrated in {numref}`f:aperture`B. 
+Between vertex points, the aperture can can follow a straight line or the arc of an ellipse. 
+The vertex points are specified by setting the `wall2d` parameter. This parameter has three
+subcomponents
+```{code} yaml
+center              # (x, y) center pointer.
+absolute_vertices   # Boolian. Default is False.
+vertices            # Ordered list of vertex points.
+```
+Example:
+```{code} yaml
+Aperture:
+  wall2d:
+    center: [-0.045, 0.011]
+    absolute_vertices: True
+    vertices:
+      - point: [0.023, 0.069]      # V1
+      - point: [-0.025, 0.067]     # V2
+      - radius: [0.08, 0.04]
+      - tilt: 0.12
+      - point: [-0.088, 0.036]     # V3
+      - point: [-0.088, -0.021]    # V4
+      - point: [0.023, -0.033]     # V5
+```
+This corresponds roughly to what is shown in {numref}`f:aperture`.
+
+If the boolean `absolute_vertices` is set `False`, which is the default,
+the vertex point positions are with respect to the `center` point. 
+That is, the vertex point positions in absolute terms are the positions given with each vertex plus
+the position of the `center`. If `absolute_vertices` is `True`, the vertex point positions 
+are independent of the `center` point. The default position of the `center` is the origin.
+
+The `vertices` component of `wall2d` contains an ordered  list of vertex 
+`point`s with each one specifying an {math}`(x, y)` position. 
+In between any two `point`s, there can optionally be a single `radius` element.
+If there is a `radius` element, there can also be a  single `tilt` element between
+the `point`s but `tilt` can only be present if there is a `radius` present.
+There can also be a `radius` element and `tilt` element after the last `point` in the `vertices`list. 
+If these are present, they are considered to be between the last and fist `point`s in the list.
+
+The aperture is constructed by connecting consecutive `point`s in the `vertices` list along with
+connecting the last `point` to the first. If there is no `radius` between consecutive `point`s,
+the aperture follows a straight line between the points. If there is a `radius` element between
+two points, the aperture is either a circular arc or a section of an ellipse. 
+A `radius` can have either a single value or a list of two values. If a `radius` has a single
+value, this is the radius of the circular arc connecting the vertices. In this case, the `tilt`
+element may not be present. If the `radius` has two values, these are the half lengths of the 
+principal diameters. If the `tilt` is zero (the default), the ellipse is upright with the first 
```

Comment:
Recommend specifying the units of `tilt` here (e.g., deg or rad).

Reply:
Again are the units of all quantities be specified even though they are specified in the Introduction?

Reply:
Same comment as above.

Reply:
Added units info.